app/utils/constants.ts

```diff
@@ -105,8 +105,22 @@ const PROVIDER_LIST: ProviderInfo[] = [
       { name: 'mistral-small-latest', label: 'Mistral Small', provider: 'Mistral' },
       { name: 'codestral-latest', label: 'Codestral', provider: 'Mistral' },
       { name: 'mistral-large-latest', label: 'Mistral Large Latest', provider: 'Mistral' }
+      { name: 'pixtral-12b-2409', label: 'Pixtral 12B', provider: 'Mistral' },
     ],
     getApiKeyLink: 'https://console.mistral.ai/api-keys/'
+  }, {
+    name: 'Cerebras',
+    staticModels: [
+      { name: 'llama3.1-8b', label: 'Llama 3.1 8B', provider: 'Cerebras' },
+      { name: 'llama3.1-70b', label: 'Llama 3.1 70B', provider: 'Cerebras' }
+    ],
+    getApiKeyLink: 'https://inference-docs.cerebras.ai/quickstart'
+  }, {
+    name: 'Bedrock',
+    staticModels: [
+      { name: 'anthropic.claude-3-5-sonnet-20241022-v2:0', label: 'Claude 3.5 Sonnet v2', provider: 'Bedrock' },
+    ],
+    getApiKeyLink: 'https://platform.deepseek.com/api_keys'
```

Comment:
Not sure but I would probably link to this
https://docs.aws.amazon.com/bedrock/latest/userguide/getting-started-api.html

Reply:
Oh that's a mistake, I forgot to change the url when I was merging changes manually after syncing my fork. I'll correct that asap